README.md

```diff
@@ -1,32 +1,27 @@
-# Setup-Portfolio Mobile Version Project
+# Setup-Portfolio Desktop Version
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 ​
-Showcasing my portfolio through esctatic web design project.
+Showcasing my web design ability through esctatic web design.
 ​
 ## Built With
 ​
 - HTML 
 - CSS
 - Linters
 ​
+​
 ## Getting Started
-
-To get a local copy up and running follow these simple steps:
-
-- copy this link `https://github.com/Epaltechs/Setup-Portfolio.git`.
-- get the directory that you want to clone the repository.
-- open the command prompt in this directory.
-- write `git clone https://github.com/ShoiraTa/blog_app.git`.
-- go to the repository folder in your command prompt `cd Setup-Portfolio Mobile Version`
-
-
+​
+To get a local copy up and running follow these simple example steps.
+​
 ## Authors
-
-👤 **Emmnanuel Paul**
-- GitHub: [@Epaltechs](https://github.com/Epaltechs)
-- Twitter: [@Emmnanuel](https://twitter.com/@epaltechs) 
-- LinkedIn: [emmanuel-paul)](https://www.linkedin.com/in/emmanuel-paul-a2bab7b4/)
-
+​
+:bust_in_silhouette: **Author**
+- Emmnanuel Paul
+- GitHub: [epaltechs.github.io/Setup-Portfolio]
+- LinkedIn: [https://www.linkedin.com/in/emmanuel-paul-a2bab7b4/]
+= Twitter: [http://twitter.com/@emmapaul247].
+​
 ## :handshake: Contributing
```

Comment:
- [ ] Kindly provide a professional README file by adding all required sections. Your README should contain all possible sections in the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md).